frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -291,45 +358,65 @@ const LandForm: React.FC<IParentLandForm> = (props: IParentLandForm) => {
     return response?.available;
   };
 
+  const steps: IStep[] = [
+    {
+      route: 'identification',
+      title: 'Parcel ID',
+      completed: false,
+      canGoToStep: true,
+      validation: { schema: LandIdentificationSchema, nameSpace: () => 'data' },
+    },
+    {
+      route: 'usage',
+      title: 'Usage',
+      completed: false,
+      canGoToStep: !!props.disabled,
+      validation: { schema: LandUsageSchema, nameSpace: () => 'data' },
+    },
+    {
+      route: 'valuation',
+      title: 'Valuation',
+      completed: false,
+      canGoToStep: !!props.disabled,
+      validation: { schema: LandValuationSchema, nameSpace: () => 'data' },
+    },
+  ];
+
+  if (!!props.initialValues?.buildings?.length) {
+    steps.push({
+      route: 'associatedLand',
+      title: 'View Associated Land',
+      completed: false,
+      canGoToStep: !!props.disabled,
+    });
+  }
+  steps.push({
+    route: 'review',
+    title: 'Review',
+    completed: false,
+    canGoToStep: !!props.disabled,
+    validation: { schema: ParcelSchema, nameSpace: () => 'data' },
+  });
+
   return (
     <Container className="landForm">
       <SteppedForm
         // Provide the steps
-        steps={[
-          {
-            route: 'identification',
-            title: 'Parcel ID',
-            completed: false,
-            canGoToStep: true,
-            validation: { schema: LandIdentificationSchema, nameSpace: () => 'data' },
-          },
-          {
-            route: 'usage',
-            title: 'Usage',
-            completed: false,
-            canGoToStep: false,
-            validation: { schema: LandUsageSchema, nameSpace: () => 'data' },
-          },
-          {
-            route: 'valuation',
-            title: 'Valuation',
-            completed: false,
-            canGoToStep: false,
-            validation: { schema: LandValuationSchema, nameSpace: () => 'data' },
-          },
-          {
-            route: 'review',
-            title: 'Review',
-            completed: false,
-            canGoToStep: false,
-            validation: { schema: ParcelSchema, nameSpace: () => 'data' },
-          },
-        ]}
-        persistable={true}
+        steps={steps}
+        persistable={!props.disabled}
         persistProps={{
           name: 'land',
           secret: keycloak.obj.subject,
-          persistCallback: noop,
+          persistCallback: (values: ISteppedFormValues<IFormParcel>) => {
```

Comment:
if the formik persist callback is triggered (meaning a draft was loaded) reset the tabs to default but load the saved data into the initial values state. This ensures the form updates the stepper to properly display the steps of the new data.